Sources/Database/Live.swift

```diff
@@ -30,10 +31,13 @@ extension Client {
           RETURNING *
           """
         )
-        .first(decoding: EnterpriseAccount.self)
+        .first()
+        .get()
+        .unwrap()
+        .decode(model: EnterpriseAccount.self, keyDecodingStrategy: .convertFromSnakeCase)
```

Comment:
For some reason NIO Postgres doesn't have a way to configure this for the entire pool, so a lot of these lines are verbose.

We had `EitherIO` helpers that hid away the work and we could bring some similar helpers back in the future, but wanted to finish the conversion first.